web/src/components/FilteredConnection.tsx

```diff
@@ -404,11 +441,27 @@ export class FilteredConnection<N, NP = {}, C extends Connection<N> = Connection
         super(props)
 
         const q = new URLSearchParams(this.props.location.search)
+
+        // Note: Do not set `after` from the URL, as this doesn't track the number
```

Comment:
Nit: this rewording might make the comment block a little bit clearer (otherwise it's not fully obvious how it relates to the code around it).

```suggestion
        // Note: in the initial state, Do not set `after` from the URL, as this doesn't track the number
```

Reply:
Updated.